.github/dependabot.yml

```diff
@@ -3,6 +3,6 @@ updates:
 - package-ecosystem: pip
   directory: /
   schedule:
-    interval: daily
+    interval: monthly
```

Comment:
Looks like even weekly is supported
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval

Have you checked "[Grouping Dependabot updates into one pull request](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/customizing-dependency-updates#grouping-dependabot-updates-into-one-pull-request)"

There seems to be similar for security updates
https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates#about-grouped-security-updates

Reply:
Thanks!

I'll see if we can group some deps and then set it as weekly

Reply:
Agreed!

I still need time to properly configure it in the way it works weekly with groups and everything.

I'll open similar PRs for other 2 projects, please ack here and there as well once opened, let's have a pacefull weekend 😄

Reply:
There:

- https://github.com/release-engineering/pubtools-marketplacesvm/pull/33
- https://github.com/release-engineering/cloudpub/pull/32